doc/js_tutorials/js_assets/js_canny.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Image Canny Example</title>
+<link href="js_example_style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+<h2>Image Canny Example</h2>
+<p>
+    Canvas elements named <b>canvasInput</b> and <b>canvasOutput</b> have been prepared.
+    Choose an image and click <b>Try it</b> button to see the result.
```

Comment:
Highlight `Try it` by saying "Click <b>Try it</b> button to see the result. You can choose another image."